content/en/blog/2024/spring-starter-stable/index.md

```diff
@@ -0,0 +1,692 @@
+
+We are going to add an [OpenTelemetry collector](https://opentelemetry.io/docs/collector/) and display the telemetry data in the collector logs.
+
+You can also use your favorite OTLP-compatible backend - we use collector logs for simplicity here.
+
+To do this, let's add the following `docker-compose-otel.yml` and `collector-spring-native-config.yaml` files in the `spring-petclinic` directory: 
+
+```yaml
+version: '3.8'
+services:
+  app:
+    image: spring-petclinic-native
+    environment:
+      OTEL_SERVICE_NAME: "graal-native-example-app"
+      OTEL_EXPORTER_OTLP_ENDPOINT: "http://collector:4318"
+    ports:
+      - "8080:8080"
+    depends_on:
+      - collector
+  collector:
+    image: otel/opentelemetry-collector-contrib:0.109.0
+    volumes:
+      - ./collector-spring-native-config.yaml:/collector-spring-native-config.yaml
+    command: ["--config=/collector-spring-native-config.yaml"]
+    expose:
+      - "4317"
+    ports:
+      - "4317:4317"
+```
+
+collector-spring-native-config.yaml
+```yaml 
+receivers:
+  otlp:
+    protocols:
+      http:
+        endpoint: "0.0.0.0:4318"
+exporters:
+  debug:
+    verbosity: detailed
+service:
+  pipelines:
+    metrics:
+      receivers: [otlp]
+      exporters: [debug]
+    traces:
+      receivers: [otlp]
+      exporters: [debug]
+    logs:
+      receivers: [otlp]
+      exporters: [debug]
+
+```
+
+Now, we can run the Spring PetClinic application and the OpenTelemetry collector:
+
+```bash
+docker-compose -f docker-compose-otel.yml up
+```
+
+Let's check the collector logs to see the telemetry data.
+
+We can spot one log record about the Spring PetClinic application startup:
+
+```
+2024-09-16 14:19:11 collector-1  | LogRecord #2
+2024-09-16 14:19:11 collector-1  | ObservedTimestamp: 2024-09-16 12:19:10.38137 +0000 UTC
+2024-09-16 14:19:11 collector-1  | Timestamp: 2024-09-16 12:19:10.379 +0000 UTC
+2024-09-16 14:19:11 collector-1  | SeverityText: INFO
+2024-09-16 14:19:11 collector-1  | SeverityNumber: Info(9)
+2024-09-16 14:19:11 collector-1  | Body: Str(Started PetClinicApplication in 0.489 seconds (process running for 0.493))
+```
+
+The OpenTelemetry Spring Boot starter has instrumented Logback and has sent the 'Started PetClinicApplication in 0.489 seconds (process running for 0.493)' telemetry log record to the OpenTelemetry collector.
+
+Let's open the <http://localhost:8080/vets.html> in our web browser or execute the following curl command:
+
+```shell
+curl http://localhost:8080/vets.html
+```
+
+If we look at the collector logs, we can see that one span has been created for the HTTP request with the Trace ID `16a0a5be5127309858c7c63a76b3f471` (it will be a different Trace ID in your case):
```

Comment:
```suggestion
If we look at the Collector logs, we can see that one span has been created for the HTTP request with the Trace ID `16a0a5be5127309858c7c63a76b3f471` (it will be a different Trace ID in your case):
```